flytekit/core/task.py

```diff
@@ -130,6 +130,7 @@ def task(
     pod_template: Optional["PodTemplate"] = ...,
     pod_template_name: Optional[str] = ...,
     accelerator: Optional[BaseAccelerator] = ...,
+    unsafe: bool = ...,
```

Comment:
I prefer to have a more explicit name. Something like:

```suggestion
    pickle_untyped: bool = ...,
```